hangman/test/hangman_test.exs

```diff
@@ -0,0 +1,69 @@
+defmodule HangmanTest do
```

Comment:
## +5 thank you for having these tests

A couple of suggestions:

1. Layout is just as important in tests as in regular code,

2. There's no test for the already_used state